mon-pix/tests/unit/mixins/secured-route-mixin-test.js

```diff
@@ -0,0 +1,166 @@
+
+describe.only('Unit | Mixin | secured-route-mixin', function() {
+  setupTest();
+
+  let sinon;
+  let route;
+  let router;
+  let transition;
+
+  beforeEach(function() {
+    sinon = sinonjs.createSandbox();
+  });
+
+  afterEach(function() {
+    sinon.restore();
+  });
+
+  describe('#beforeModel', function() {
+    beforeEach(function() {
+      const MixinImplementingBeforeModel = Mixin.create({
+        beforeModel() {
+          return RSVP.resolve('upstreamReturnValue');
+        }
+      });
+
+      transition = {
+        intent: {
+          url: '/transition/target/url'
+        },
+        send() {}
+      };
+      this.owner.register('service:router', Service.extend({
+        transitionTo() {}
+      }));
+      router = this.owner.lookup('service:router');
+
+      this.owner.register('service:session', Service.extend());
+      this.owner.register('service:currentUser', Service.extend({
+        user: {}
+      }));
+
+      route = Route.extend(MixinImplementingBeforeModel, SecuredRouteMixin).create();
+      setOwner(route, this.owner);
+
+      sinon.spy(transition, 'send');
+      sinon.spy(router, 'transitionTo');
+    });
+
+    describe('when the session is authenticated', function() {
+
+      beforeEach(function() {
+        const session = this.owner.lookup('service:session');
+        session.set('isAuthenticated', true);
+      });
+
+      describe('when the user has accepted the terms of service', function() {
+
+        beforeEach(function() {
+          const currentUser = this.owner.lookup('service:currentUser');
+          currentUser.user.mustValidateTermsOfService = false;
+        });
+
+        it('returns the upstream promise', async function() {
+          const result = await route.beforeModel(transition);
+
+          expect(result).to.equal('upstreamReturnValue');
+        });
+
+        it('does not transition to the authentication route', function() {
+          route.beforeModel(transition);
+
+          sinon.assert.neverCalledWith(router.transitionTo, 'login');
+        });
+
+        it('does not transition to the terms-of-service route', function() {
+          route.beforeModel(transition);
+
+          sinon.assert.neverCalledWith(router.transitionTo, 'terms-of-service');
+        });
+      });
+
+      describe('when the user has not accepted the terms of service', function() {
+
+        beforeEach(function() {
+          const currentUser = this.owner.lookup('service:currentUser');
+          currentUser.user.mustValidateTermsOfService = true;
+        });
+
+        it('does not returns the upstream promise', async function() {
+          expect(route.beforeModel(transition)).to.be.undefined;
+        });
+
+        it('does not transition to the authentication route', function() {
+          route.beforeModel(transition);
+
+          sinon.assert.neverCalledWith(router.transitionTo, 'login');
+        });
+
+        it('does transition to the terms-of-service route', function() {
```

Comment:
```suggestion
        it('transitions to  terms-of-service route', function() {
```
Le verbe transition [existe](https://en.wiktionary.org/wiki/transition#English), et il prend un s, [classique](https://conjugator.reverso.net/conjugation-english-verb-transition.html)

Reply:
J'ai hésité avant de mettre ce commentaire, en me disant que j'allais passer pour un puriste. Allez, j'assume !